test/api/test_knowledge_graph.py

```diff
@@ -139,7 +139,120 @@ def test_knowledge_graph_one_hop(query_options):
     assert "datetime" in resp_json["return value"]
 
 
+@pytest.mark.parametrize("query_options", kg_options)
+@load_data(APP, """
+    PatientId,year,AgeStudyStart,Albuterol,AvgDailyPM2.5Exposure
+    varchar(255),int,varchar(255),varchar(255),int
+    1,2010,0-2,0,1
+    2,2010,0-2,1,1
+    3,2010,0-2,>1,1
+    4,2010,0-2,0,2
+    5,2010,0-2,1,2
+    6,2010,0-2,>1,2
+    7,2010,0-2,0,3
+    8,2010,0-2,1,3
+    9,2010,0-2,>1,3
+    10,2010,0-2,0,4
+    11,2010,0-2,1,4
+    12,2010,0-2,>1,4
+""")
+def test_knowledge_graph_one_hop_valid_trapi_1_1_response(query_options):
+    """Test one-hop return valid TRAPI 1.1 response."""
```

Comment:
We could use [reasoner-validator](https://github.com/NCATSTranslator/reasoner-validator) for this purpose.

Reply:
just created a new branch to do that.